lib/logatron/const.rb

```diff
@@ -1,15 +1,19 @@
 module Logatron
-  WARN = 'WARN'
-  INFO = 'INFO'
-  ERROR = 'ERROR'
-  CRITICAL = 'CRITICAL'
-  FATAL = 'FATAL'
-  DEBUG = 'DEBUG'
   MSG_ID = :msg_id
   SITE = :site
+
+  DEBUG = 'DEBUG'.freeze
+  INVALID_USE = 'INVALID_USE'.freeze
+  INFO = 'INFO'.freeze
+  WARN = 'WARN'.freeze
+  ERROR = 'ERROR'.freeze
+  CRITICAL = 'CRITICAL'.freeze
+  FATAL = 'FATAL'.freeze
+
   SEVERITY_MAP =  {
       Logatron::DEBUG => 0,
       Logatron::INFO => 1,
+      Logatron::INVALID_USE => 1,
```

Comment:
I'll make an entry into the readme - now that we know how the integer is used I can better articulate it